src/menu/src/Menu.js

```diff
@@ -101,9 +104,19 @@ const Menu = memo(function Menu(props) {
   }, [menuRef])
```

Comment:
Given that menuRef wont actually trigger this hook to run again when it changed (mutable refs wont trigger renders or hooks to run again), I would opt for us calculating `isEmpty` separately, on every render. We don't need useState for it.

Reply:
Ok, I got it. Once I do it I will do the PR.